src/snowflake/snowpark/table.py

```diff
@@ -658,12 +680,15 @@ def merge(
             else result
         )
 
-    def drop_table(self) -> None:
+    def drop_table(self, if_exists: bool = False) -> None:
```

Comment:
@sfc-gh-stan , we don't have this the main branch, is this for testing purpose and if so shall we remove this?

Reply:
gonna remove it in this PR

Reply:
removed